apps/dashboard/app/views/layouts/nav/_group.html.erb

```diff
@@ -1,27 +1,28 @@
-<li class="dropdown" title="<%= group.title %>" >
-  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><%= group.title %><span class="caret"></span></a>
-  <ul class="dropdown-menu">
+<li class="nav-item dropdown" title="<%= group.title %>">
+  <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><%= group.title %><span class="caret"></span></a>
+
+  <ul class="dropdown-menu" role="menu">
     <% OodAppGroup.groups_for(apps: group.apps, group_by: :subcategory).each_with_index do |g, index| %>
-      <%= content_tag "li", "", class: "divider", role: "separator" if index > 0 %>
-      <%= content_tag "li", g.title, class: "dropdown-header" unless g.title.empty? %>
+      <%= content_tag(:li, nil, class: ["dropdown-divider"], role: "separator") if index > 0 %>
+      <%= content_tag(:li, g.title, class: ["dropdown-header"]) unless g.title.empty? %>
       <% g.apps.each do |app| %>
         <% app.links.each do |link| %>
-          <li>
```

Comment:
Shouldn't remove the `li` here. Visually it's the same, but the links need to be in list items for accessibility.